packages/elements/src/interactive-chart/__test__/interactive-chart.test.js

```diff
@@ -534,5 +534,36 @@ describe('interactive-chart/InteractiveChart', function () {
       expect(legendStyle.position).to.equal('absolute');
       expect(legendLeftPosition).to.equal(InteractiveChart.DEFAULT_LEGEND_LEFT_POSITION);
     });
+
+    it('Should render new Legend when add new seriesList.', async function () {
+      const data = [
+        {
+          time: 1678147200,
+          open: 5679,
+          high: 5694,
+          low: 5544,
+          close: 5547,
+          value: 5547
+        }
+      ];
+      el.config = {
+        series: [
+          {
+            symbol: 'Price',
+            type: 'candlestick'
+          }
+        ]
+      };
+      await elementUpdated(el);
+      await nextFrame(3); // wait for resize observer & rendering completion
+      expect(el.hasDataPoint).to.be.false;
+
+      await nextFrame();
+      el.seriesList[0].setData(data);
+      el.config.series[0].data = data;
+      await elementUpdated(el);
+      await nextFrame(3); // wait for resize observer & rendering completion
+      expect(el.hasDataPoint).to.be.true;
```

Comment:
I think it's better to add expect point on rowLegend. Like this.
```javascript
expect($0.rowLegend[0].querySelector('.price').textContent).to.not.equal('N/A');
```
What do you think?

Reply:
I don't like html manual query method hahaha. check `hasDataPoint` is more specific to the root cause I think.
moreover I think it should compare with 'object Object' not 'N/A'